test/conformance/tests/test_policy_configuration.go

```diff
@@ -473,17 +473,15 @@ var _ = Describe("[sriov] operator", Ordered, func() {
 					testNode = sriovInfos.Nodes[0]
 					sriovDeviceList, err := sriovInfos.FindSriovDevices(testNode)
 					Expect(err).ToNot(HaveOccurred())
-					unusedSriovDevices, err := findUnusedSriovDevices(testNode, sriovDeviceList)
+					noCarrierSriovDevices, err := findNoCarrierSriovDevices(testNode, sriovDeviceList)
```

Comment:
do we also require it to also be unsused?

Reply:
An "unused device", in the context of `findUnusedSriovDevices`, means it is not the default route, and it is not assegned to an ovs bridge as master (what happen in OVNk).

https://github.com/zeeke/sriov-network-operator-1/blob/90b11c8214118ba6177e322db0a041fd553b48cf/test/conformance/tests/test_sriov_operator.go#L1649

So, if the device has no-carrier, it is also "unused".
(Maybe it can be the uplink of a test ovs bridge? 🤔 )